components/ai_chat/resources/page/components/sidebar_nav/index.tsx

```diff
@@ -0,0 +1,186 @@
+    event.preventDefault()
+    props.onSubmit?.(value)
+  }
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Escape') {
+      props.onBlur?.()
+    }
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        className={styles.simpleInput}
+        type='text'
+        value={value}
+        onChange={handleChange}
+        autoFocus
+        onBlur={props.onBlur}
+        onKeyDown={handleKeyDown}
+      />
+    </form>
+  )
+}
+
+interface DisplayTitleProps {
+  title: string
+  description?: string
+  onEditTitle?: () => void
+  onDelete?: () => void
+}
+
+function DisplayTitle(props: DisplayTitleProps) {
+  const [isButtonMenuVisible, setIsButtonMenuVisible] = React.useState(false)
+
+  return (
+    <div
+      className={styles.displayTitle}
+      onMouseEnter={() => setIsButtonMenuVisible(true)}
+      onMouseLeave={() => setIsButtonMenuVisible(false)}
+    >
+      <div className={styles.displayTitleContent}>
+        <div
+          className={styles.text}
+          onDoubleClick={props.onEditTitle}
+        >
+          {props.title}
+        </div>
+        <div className={styles.description}>{props.description}</div>
+      </div>
+      {isButtonMenuVisible && (
+        <ButtonMenu className={styles.optionsMenu}>
+          <div
+            slot='anchor-content'
+            className={styles.optionsButton}
+          >
+            <Icon name='more-vertical' />
+          </div>
+          <leo-menu-item onClick={props.onEditTitle}>
+            <div className={styles.optionsMenuItemWithIcon}>
+              <Icon name='edit-pencil' />
+              <div>{getLocale('menuRenameConversation')}</div>
+            </div>
+          </leo-menu-item>
+          <leo-menu-item onClick={props.onDelete}>
+            <div className={styles.optionsMenuItemWithIcon}>
+              <Icon name='trash' />
+              <div>{getLocale('menuDeleteConversation')}</div>
+            </div>
+          </leo-menu-item>
+        </ButtonMenu>
+      )}
+    </div>
+  )
+}
+
+interface SidebarNavProps {
+  enableBackButton?: boolean
+  setIsConversationListOpen: (value: boolean) => unknown
+}
+
+export default function SidebarNav(props: SidebarNavProps) {
+  const aiChatContext = useAIChat()
+  const conversationContext = useConversation()
+
+  return (
+    <>
+      <div
+        className={classnames({
+          [styles.header]: true,
+          [styles.noBorder]: !props.enableBackButton
+        })}
+      >
+        {!aiChatContext.isStandalone && props.enableBackButton && (
+          <Button
+            kind='plain-faint'
+            fab
+            onClick={() => {
+              props.setIsConversationListOpen?.(false)
+            }}
+          >
+            <Icon name='arrow-left' />
+          </Button>
+        )}
+        <form className={styles.form}>
+          <input
+            placeholder='Search your conversations'
```

Comment:
nit: translate

Reply:
oh actually we need to completely remove search